assets/src/dashboard/app/views/templateDetails/index.js

```diff
@@ -240,7 +277,13 @@ function TemplateDetails() {
   );
 
   if (!template) {
-    return null;
+    return (
+      <Layout.Provider>
+        <Layout.Fixed>
+          <DetailViewNavBar handleCta={onHandleCta} />
```

Comment:
We will probably want a more detailed error view, but for now it has a toast + close button so users aren't stuck in the app.